mypy/test/test_check.py

```diff
@@ -209,3 +206,5 @@ def parse_flags(self, program_text: str) -> List[str]:
             return m.group(1).split()
         else:
             return []
+
+TestTypeCheck.setup_tests()
```

Comment:
Hmm, typically the magic of pytest makes it so you don't need to make this kind of call up at toplevel. Is there another way we can prepare these?

Also, "setup" in the pytest context usually refers to something you do just before running some tests in order to set up some state that the test itself, or the application code it's testing, expect in order for the test to run. It's not normally something you'd want to do at toplevel like this. Is "collect" or "test collection" more what this step is about?